src/goto-instrument/contracts/contracts.cpp

```diff
@@ -187,26 +187,7 @@ void code_contractst::check_apply_loop_contracts(
 
```

Comment:
Assigns clause is always optional (since we try to infer it), so no message is shown regarding that. I added a message regarding missing decreases clause -- that termination will not be verified.